src/style-spec/reference/v8.json

```diff
@@ -6061,6 +6061,82 @@
       },
       "property-type": "data-constant"
     },
+    "raster-color": {
+      "type": "color",
+      "doc": "Defines a color map applied to a raster layer. Should be an expression that uses `[\"raster-value\"]` as input.",
+      "transition": false,
+      "sdk-support": {
+        "basic functionality": {
+          "js": "2.12.0"
+        },
+        "data-driven styling": {}
+      },
+      "expression": {
+        "interpolated": true,
+        "parameters": [
+          "raster-value"
+        ]
+      },
+      "property-type": "color-ramp"
+    },
+    "raster-color-mix": {
+      "type": "array",
+      "default": [
+        0.2126,
+        0.7152,
+        0.0722,
+        0.0
+      ],
+      "length": 4,
+      "value": "number",
+      "property-type": "data-constant",
+      "transition": true,
+      "expression": {
+        "interpolated": true,
+        "parameters": [
+          "zoom"
+        ]
+      },
+      "doc": "The combination of source RGB channels added together to compute the raster value. Computed using the equation `mix.r * src.r + mix.g * src.g + mix.b * src.b + mix.a`. Source alpha is applied as opacity of the output, while ",
```

Comment:
Oops. This is unfinished. These parameter docs need to be thought out and described more completely.